iocore/net/quic/QUICLossDetector.h

```diff
@@ -46,6 +48,18 @@ struct PacketInfo {
   QUICPacketUPtr packet;
 };
 
+class QUICRTTProvider
+{
+public:
+  ink_hrtime smoothed_rtt() const;
+
+protected:
+  ink_hrtime _latest_rtt   = 0;
```

Comment:
Can you tell me why these members are moved RTTProvider?

I'd prefer keep the variables in LossDetector because 1) these are variables defined on the spec with other variables, 2) and RTTProvider doesn't use/need three of them.
https://tools.ietf.org/html/draft-ietf-quic-recovery-10#section-3.4.2

IMO, `QUICRTTProvider::smoothed_rtt()` should be pure virtual function.